packages/backend/src/tokens/interstitialRule.ts

```diff
@@ -87,7 +88,17 @@ export const isNormalSignedOutState: InterstitialRule = async options => {
 
 // This happens when a signed in user visits a new subdomain for the first time. The uat will be available because it's set on naked domain, but session will be missing. It can also happen if the cookieToken is manually removed during development.
 export const hasClientUatButCookieIsMissingInProd: InterstitialRule = async options => {
-  const { clientUat, cookieToken } = options as any;
+  const {
+    clientUat,
+    isSatellite,
+    cookieToken,
+    isSynced = false,
+  } = options as AuthenticateRequestOptionsWithExperimental;
+
+  if (clientUat && !cookieToken && isSatellite && isSynced) {
```

Comment:
```suggestion
  if (clientUat && Number.parseInt(clientUat) > 0 && !cookieToken && isSatellite && isSynced) {
```

At this point in the flow, we've already thrown the "signed out" state to anyone with `clientUat==='0'`, so this change is only meant to help with readability.

I know that lower in this function it also only checks `clientUat && !cookieToken`... I'd love if you added the `> 0` check there too!